napi.h

```diff
@@ -1820,6 +1821,193 @@ namespace Napi {
     bool _suppress_destruct;
   };
 
+  class ThreadSafeFunction {
+  public:
+    // Default set
```

Comment:
What does "Default set" mean?

Also, we should have comments that split the API into functions which can only be called from the main thread, and those that are thread-safe.

Reply:
> What does "Default set" mean?

Unmeaningful. There are many overloaded methods. So, I was so confusing. To clarify it during development, it's just used for my convenience.

> Also, we should have comments that split the API into functions which can only be called from the main thread, and those that are thread-safe.

Done